bindings/python/meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'tree-sitter-capnp',
+  'c',
+  license: 'MIT'
+)
+
+python = import('python').find_installation('python3', modules: ['cffi'], pure: false)
+python_dep = python.dependency(required: true)
+
+ts_cffi_srcs = configure_file(
+  command: [python, files('ffibuilder.py'), '@OUTPUT@'],
+  output: '_tree_sitter_capnp.c',
+)
+
+ts_pyext = python.extension_module(
+    '_tree_sitter_capnp',
+    ts_cffi_srcs,
+    files('..' / '..' / 'src' / 'parser.c'),
+    # if you have a scanner, uncomment next line
+    # files('..' / '..' / 'src' / 'scanner.c'),
+    dependencies: [python_dep],
+    install: true,
+    subdir: 'tree_sitter_capnp',
+)
+
+python.install_sources(files('tree_sitter_capnp' / '__init__.py'), subdir: 'tree_sitter_capnp')
```

Comment:
The main value of files() is for obtaining cross-directory references saved as reusable variables, that remember the directory in which they were defined. Passing strings to install_sources() is also valid, and internally converts immediately to a `mesonbuild.mesonlib.File` object.

(The second value of it is for cases where a function accepts command strings, not "list of source files", and you need to make sure meson understands it is a file, not "just" an argument to a command.)

It is slightly shorter, and identical in effect, so why not do a bit less typing. :D

Another thing available since 0.64.0:

```meson
python.install_sources('tree_sitter_capnp/__init__.py'), preserve_path: true)
```

will install the entire structure as defined, so if you installed the "tree_sitter_capnp/__init__.py" file it will install the python module "tree_sitter_capnp" subdir's file "__init__.py".